### ISSUE TITLE
Disable Isolation by End of March

### ISSUE THREAD

**Issue by maennchen** [jshmrtn/hygeia#1340]
## Is your feature request related to a problem? Please describe.

The government no longer regulates isolations starting from April.

## Describe the solution you'd like

* Disable AT Invite in Inbox
* Rest works as before
* Manual Query to end running isolations

## Describe alternatives you've considered

None

## Additional context

None

**Comment by maennchen** [comment]
@rebecca-br / @saanni / @St-Annette Bereit zum testen. (Details via Email an Rebecca.)

**Comment by rebecca-br** [comment]
@maennchen wie soll ich das testen? Also beim Import schauen, dass das AT nicht ausgelöst wird?

**Comment by maennchen** [comment]
@rebecca-br Genau. Es hatt auch zusätzlich noch einen kleinen roten hinweis beim Import.

**Comment by rebecca-br** [comment]
@maennchen funktioniert. :-) und der Hinweis wird angezeigt. 

<img width="763" alt="Bildschirmfoto 2022-04-01 um 11 31 01" src="https://user-images.githubusercontent.com/93326696/161236940-bf1a77d6-cdd5-4c9c-8436-945de9e55e91.png">
